src/code/z_bgcheck.c

```diff
@@ -1947,13 +1912,14 @@ s32 BgCheck_CheckWallImpl(CollisionContext* colCtx, u16 xpFlags, Vec3f* posResul
     dz = posNext->z - posPrev->z;
 
     // if there's movement on the xz plane, and argA flag is 0,
-    if (((dx != 0.0f) || (dz != 0.0f)) && !(argA & 1)) {
-        if ((checkHeight + dy) < 5.0f) {
+    if ((dx != 0.0f || dz != 0.0f) && !(argA & 1)) {
+        if (checkHeight + dy < 5.0f) {
```

Comment:
```suggestion
    if (((dx != 0.0f) || (dz != 0.0f)) && !(argA & 1)) {
        if ((checkHeight + dy) < 5.0f) {
```
These brackets were dropped but we're trying to preserve brackets